query/src/exec/context.rs

```diff
@@ -254,6 +254,8 @@ impl IOxExecutionContext {
         let ctx = self.child_ctx("prepare_sql");
         debug!(text=%sql, "planning SQL query");
         let logical_plan = ctx.inner.create_logical_plan(sql)?;
+        let s = logical_plan.display_graphviz().to_string();
+        debug!(plan=?s, "logical plan");
```

Comment:
```suggestion
        debug!(plan=%logical_plan.display_graphviz(), "logical plan");
```

I think as written the plan string will be created each time, not just when debug logging is created. This way would only print the plan when debug logging is on

Reply:
Good catch :)